google/cloud/firestore_v1/base_query.py

```diff
@@ -847,7 +849,7 @@ def _enum_from_op_string(op_string) -> Any:
 
     Args:
         op_string (str): A comparison operation in the form of a string.
-            Acceptable values are ``<``, ``<=``, ``==``, ``>=``
+            Acceptable values are ``<``, ``<=``, ``==``, ``!=``, ``>=``
```

Comment:
does this method also accept "not-in, array_contains" and more?

Reply:
@crwilcox  Yes, this method accept all the values  including `array_contains` we don't have any special condition for that, limitation of acceptable values are just mentioned in doc-string.
Is this question related to https://github.com/googleapis/python-firestore/issues/203 issue?